backside/spots/dto/spot.go

```diff
@@ -0,0 +1,10 @@
+package dto
+
+import "sk8.town/backside/errs"
+
+func (x *SpotRequest) Validate() *errs.AppError {
+	if x.Coordinates == nil {
+		return errs.NewValidationError(`Coordinates not provided`)
+	}
+	return nil
+}
```

Comment:
We need to validate other fields. You can skip this for now and we will tackle it in a separate issue. I'm thinking if we should use this: https://github.com/go-validator/validator#usage

Reply:
issue created: https://github.com/sk8mate/sk8/issues/49